### ISSUE TITLE
Uncaught TypeError: Cannot read property 'file' of undefined

### ISSUE THREAD

**Issue by wushang1987** [magbicaleman/open-in-browser#9]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.176.0
**System**: Unknown Windows Version
**Thrown From**: [open-in-browser](https://github.com/magbicaleman/open-in-browser) package, v0.4.0

### Stack Trace

Uncaught TypeError: Cannot read property 'file' of undefined

```
At /c:/Users/WUSHANG/.atom/packages/open-in-browser/lib/open-in-browser.coffee:50

TypeError: Cannot read property 'file' of undefined
  at Object.module.exports.open (c:\Users\WUSHANG\.atom\packages\open-in-browser\lib\open-in-browser.coffee:30:26)
  at atom-workspace.<anonymous> (c:\Users\WUSHANG\.atom\packages\open-in-browser\lib\open-in-browser.coffee:7:60)
  at atom-workspace.handler (c:\Users\WUSHANG\AppData\Local\atom\app-0.176.0\resources\app\src\space-pen-extensions.js:115:32)
  at atom-workspace.jQuery.event.dispatch (c:\Users\WUSHANG\AppData\Local\atom\app-0.176.0\resources\app\node_modules\space-pen\vendor\jquery.js:4681:9)
  at atom-workspace.elemData.handle (c:\Users\WUSHANG\AppData\Local\atom\app-0.176.0\resources\app\node_modules\space-pen\vendor\jquery.js:4359:46)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (c:\Users\WUSHANG\AppData\Local\atom\app-0.176.0\resources\app\src\command-registry.js:243:29)
  at c:\Users\WUSHANG\AppData\Local\atom\app-0.176.0\resources\app\src\command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (c:\Users\WUSHANG\AppData\Local\atom\app-0.176.0\resources\app\node_modules\atom-keymap\lib\keymap-manager.js:558:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (c:\Users\WUSHANG\AppData\Local\atom\app-0.176.0\resources\app\node_modules\atom-keymap\lib\keymap-manager.js:396:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (c:\Users\WUSHANG\AppData\Local\atom\app-0.176.0\resources\app\src\window-event-handler.js:170:20)

```

### Commands

```
     -0:46.5 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:33.6 core:backspace (atom-text-editor.editor.mini.is-focused)
     -0:32.7 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:31.5 core:backspace (atom-text-editor.editor.mini.is-focused)
     -0:30.4 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:27.3 core:backspace (atom-text-editor.editor.mini.is-focused)
     -0:25.8 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:19.8 editor:newline (atom-text-editor.editor)
     -0:19.5 core:paste (atom-text-editor.editor)
 27x -0:17.5 core:move-left (atom-text-editor.editor)
  2x -0:16.1 core:move-right (atom-text-editor.editor)
     -0:15.3 core:backspace (atom-text-editor.editor)
  4x -0:14.2 core:save (atom-text-editor.editor)
     -0:06.6 core:backspace (atom-text-editor.editor.mini.is-focused)
     -0:04.8 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:03.3 open-in-browser:open (atom-text-editor.editor.mini.is-focused)
```

### Config

```json
{
  "core": {}
}
```

### Installed Packages

```coffee
# User
cdnjs, v1.4.0
color-picker, v1.4.4
open-in-browser, v0.4.0

# Dev
No dev packages
```

/cc @atom/core

**Comment by magbicaleman** [comment]
Hey, I just updated to v0.4.2 give that an update and let me know if you're still getting the error. Thank you.

**Comment by xerp** [comment]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.184.0
**System**: Microsoft Windows 7 Professional 
**Thrown From**: [open-in-browser](https://github.com/magbicaleman/open-in-browser) package, v0.4.2

### Stack Trace

Uncaught TypeError: Cannot read property 'file' of undefined

```
At /C:/Users/santiago.depedro/.atom/packages/open-in-browser/lib/open-in-browser.coffee:50

TypeError: Cannot read property 'file' of undefined
  at Object.module.exports.open (C:\Users\santiago.depedro\.atom\packages\open-in-browser\lib\open-in-browser.coffee:31:26)
  at atom-workspace.<anonymous> (C:\Users\santiago.depedro\.atom\packages\open-in-browser\lib\open-in-browser.coffee:8:60)
  at atom-workspace.handler (C:\Users\santiago.depedro\Downloads\Atom\resources\app\src\space-pen-extensions.js:112:34)
  at atom-workspace.jQuery.event.dispatch (C:\Users\santiago.depedro\Downloads\Atom\resources\app\node_modules\space-pen\vendor\jquery.js:4681:9)
  at atom-workspace.elemData.handle (C:\Users\santiago.depedro\Downloads\Atom\resources\app\node_modules\space-pen\vendor\jquery.js:4359:46)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (C:\Users\santiago.depedro\Downloads\Atom\resources\app\src\command-registry.js:243:29)
  at CommandRegistry.handleCommandEvent (C:\Users\santiago.depedro\Downloads\Atom\resources\app\src\command-registry.js:3:61)
  at CommandRegistry.module.exports.CommandRegistry.dispatch (C:\Users\santiago.depedro\Downloads\Atom\resources\app\src\command-registry.js:156:19)
  at [object Object].jQuery.fn.trigger (C:\Users\santiago.depedro\Downloads\Atom\resources\app\src\space-pen-extensions.js:64:23)
  at EventEmitter.<anonymous> (C:\Users\santiago.depedro\Downloads\Atom\resources\app\src\window-event-handler.js:80:67)
  at EventEmitter.emit (events.js:119:17)
  at EventEmitter.ipc.sendSync (C:\Users\santiago.depedro\Downloads\Atom\resources\atom\renderer\api\lib\ipc.js:21:31)
  at BrowserWindow.RemoteMemberFunction [as emit] (C:\Users\santiago.depedro\Downloads\Atom\resources\atom\renderer\api\lib\remote.js:110:29)
  at ContextMenuManager.module.exports.ContextMenuManager.showForEvent (C:\Users\santiago.depedro\Downloads\Atom\resources\app\src\context-menu-manager.js:169:33)
  at HTMLDocument.<anonymous> (C:\Users\santiago.depedro\Downloads\Atom\resources\app\src\window-event-handler.js:153:33)
  at HTMLDocument.handler (C:\Users\santiago.depedro\Downloads\Atom\resources\app\src\space-pen-extensions.js:112:34)
  at HTMLDocument.jQuery.event.dispatch (C:\Users\santiago.depedro\Downloads\Atom\resources\app\node_modules\space-pen\vendor\jquery.js:4681:9)
  at HTMLDocument.elemData.handle (C:\Users\santiago.depedro\Downloads\Atom\resources\app\node_modules\space-pen\vendor\jquery.js:4359:46)

```

### Commands

```
     -7:25.1 settings-view:open (atom-text-editor.editor)
  5x -6:21.3 core:backspace (atom-text-editor.editor.mini)
     -3:45.2 core:confirm (atom-text-editor.editor.mini)
     -2:43.8 core:cut (atom-text-editor.editor.mini)
  3x -2:30.7 core:backspace (atom-text-editor.editor.mini)
  2x -2:27.7 core:confirm (atom-text-editor.editor.mini)
     -1:33.7 core:backspace (atom-text-editor.editor.mini)
     -1:23.1 core:confirm (atom-text-editor.editor.mini)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "command-logger",
      "feedback",
      "release-notes",
      "autocomplete"
    ],
    "themes": [
      "atom-dark-ui",
      "monokai"
    ]
  }
}
```

### Installed Packages

```coffee
# User
atom-beautify, v0.21.3
autoclose-html, v0.15.0
autocomplete-paths, v1.0.2
autocomplete-plus, v2.3.0
autocomplete-snippets, v1.0.1
css-color-highlight, v0.3.0
css-snippets, v0.5.0
csslint, v1.0.3
highlight-line, v0.10.1
highlight-selected, v0.9.1
html-helper, v0.2.3
language-clojure, v0.13.0
linter, v0.12.0
linter-phpcs, v0.0.12
minimap, v4.4.0
minimap-find-and-replace, v4.2.0
minimap-highlight-selected, v4.2.0
monokai, v0.12.0
open-in-browser, v0.4.2
open-last-project, v0.3.4
open-recent, v2.2.0
script, v2.18.0

# Dev
No dev packages
```

**Comment by surdu** [comment]
You can reproduce this from the context menu inside the settings page of a package, if this helps.

**Comment by magbicaleman** [comment]
Are you guys seeing this in version 0.4.3?

**Comment by magbicaleman** [comment]
@wushang1987 @surdu @xerp : Just released v0.4.6, this should fix this issue. If it does not please create a new issue. Thank you for the feedback, much appreciated.
